code-challenges.js

```diff
@@ -30,32 +47,91 @@ const temperature2 = 350
 
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
+console.log(boilingPoint(temperature1))
+console.log(boilingPoint(temperature2))
+console.log(boilingPoint(temperature3))
+// output: 
+// 42 is below boiling point
+// 350 is above boiling point
+// 212 is at boiling point
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
+// process 
+// create a function called "comboLength" that take two arrays
+const comboLength = (array1,array2) =>{
+// combine the two arrays into a variable called "combine"
+const combine = array1.concat(array2)
+
+// return the length of the "combine" variable
+return combine.length
+}
+
+
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
+console.log(comboLength(padres1984WorldSeriesRuns,padres1998WorldSeriesRuns))
 
+// output: 9
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
+// process
+// create a function called "stringReverse" that accepts a string
+const stringReverse = (data) =>{
+// convert the string into an array
+var splitLetters = data.split("")
+// reverse the order of the values in the array
+var reversedLetters = splitLetters.reverse()
+// convert the array back into a string
+var backToString = reversedLetters.join("")
+return backToString
+} 
```

Comment:
Amazing pseudocode in between the lines of code! Love the way you are breaking down the problem.